Documentation/API.md

```diff
@@ -12,6 +12,7 @@ Endpoints
 * [User Data](#section-user-data)
   * [`/v1/users/{id}`](#endpoint-user)
   * [`/v1/users/{uid}/preferences/{id}`](#endpoint-preferences)
+  * [`/v1/users/{id}/changePassword`](#endpoint-changepassword)
```

Comment:
How do you feel about the naming here?  Maybe `change_passsord` since we use underscores in all the field names.  I could also see it as just `password`.

Reply:
I'll change it to password (REST dogma says no 'verbs' in the URI!)